build/docs/build_docs.py

```diff
@@ -74,11 +74,11 @@ def main(output_dir : str, uid_base : str, package : List[str]):
     with open(output_dir / "index.md", 'w', encoding='utf8') as f:
         f.write(index_content)
 
-def format_as_section(name : str, content : Optional[str]) -> str:
+def format_as_section(name : str, content : Optional[str], heading_level : Optional[int] = 2) -> str:
```

Comment:
As is, `heading_level` looks to be an `int` and not an `Optional[int]`, since it has a default value of type `int`. Can users explicitly pass `None`?

Reply:
Ah, good catch. I added the default value here and forgot to remove the `Optional`.